packages/transaction/detail/info/transactionDetailsComponents/components.js

```diff
@@ -99,7 +108,7 @@ export const Recipient = ({ t }) => {
 export const TransactionId = ({ t }) => {
   const {
     transaction: { id },
-  } = useContext(Context);
+  } = React.useContext(TransactionDetailsContext);
```

Comment:
@eniolam1000752 @gadjacobs Please destructure `useContext` imports

Reply:
@ikem-legend for my information could you explain why it's better to be restructure?

Reply:
@soroushm `useMemo` is destructured so it's inconsistent to destructure some hooks and then use others directly with the React import. So it can be `React.useMemo` and `React.useContext` in all cases or `useMemo` and `useContext` with the first line of the file as `import React, { useMemo, useContext` } from 'react'`